src/lib/datatypes/include/sol-vector.h

```diff
@@ -166,6 +167,12 @@ sol_ptr_vector_del(struct sol_ptr_vector *pv, uint16_t i)
     return sol_vector_del(&pv->base, i);
 }
 
+static inline int
+sol_ptr_vector_del_element(struct sol_ptr_vector *pv, const void *elem)
+{
+    return sol_vector_del_element(&pv->base, elem);
```

Comment:
this is not what I'd expect, as elem in a ptr-vector is the pointed-to... you'd need to walk the vector looking for elem.